frontend/src/main/pages/HomePage.js

```diff
@@ -6,8 +6,8 @@ import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
 import CommonsList from "main/components/Commons/CommonsList";
 import { useBackend, useBackendMutation } from "main/utils/useBackend";
 import { useCurrentUser } from "main/utils/currentUser";
-import Background from './../../assets/HomePageBackground.jpg';
 import { commonsNotJoined } from "main/utils/commonsUtils";
+import getBackgroundImage from "main/components/Utils/HomePageBackground";
 
 export default function HomePage() {
```

Comment:
```suggestion
export default function HomePage({hour=null}) {
```

This suggestion goes along with the one below.